### DISCUSSION TITLE
new Blender base

### DISCUSSION THREAD

**Discussion by Reaper10** [UPBGE/upbge#1503] [Ideas/Features for UPBGE 0.3+]
This is just an idea, use  the new Blender 2.93 lts base upbge. It'll add all of this
https://www.youtube.com/watch?v=mnnMFJ3iWH0&t=17s
https://www.blender.org/download/releases/2-93/

**Comment by BluePrintRandom** [comment]
upbge is actually based on 3.0 prototyping branch already - it's easier to keep up with commits as they come in for youle.*
